packages/scenes/src/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -134,7 +134,7 @@ export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
         {width > 0 && height > 0 && (
           <PanelChrome
             title={titleInterpolated}
-            description={description ? () => model.interpolate(description) : ''}
+            description={descriptionInterpolated}
```

Comment:
```suggestion
            description={model.getDescription}
```

or some version of that, so we do not have to parse the markdown and interpolate the description string on every render, only when you hover over the info icon

Reply:
PanelChrome description prop does take a function

Reply:
Indeed, glad that you brought this becuase ..... I think we need to allow function for `description` for a viz panel as well. Overall, some core implementations report interactions, via function props (i.e. decription).